packages/odata-common/src/request-builder/action-function-import-request-builder-base.ts

```diff
@@ -27,12 +27,19 @@ export abstract class ActionFunctionImportRequestBuilderBase<
   /**
    * Execute request.
    * @param destination - Destination or DestinationFetchOptions to execute the request against.
+   * @param dataTransformer - A function to transform the received response data.
    * @returns A promise resolving to the requested return type.
    */
-  async execute(destination: DestinationOrFetchOptions): Promise<ReturnT> {
-    return this.executeRaw(destination).then(response =>
-      this.responseTransformer(response.data)
-    );
+  async execute(
+    destination: DestinationOrFetchOptions,
+    dataTransformer?: (data: any) => any
```

Comment:
Consider renaming this to `dataAccessor`, because it might otherwise be confused with what the original response transformer does.

Reply:
Good suggestion and done! I am a bit lack in creativity here. Theoretically, I would say `responseTransformer` is the best candidate, but it is used. 😆